### ISSUE TITLE
invisible IDs don't work

### ISSUE THREAD

**Issue by Killette2** [d3athrow/vgstation13#16952]
-spray ID or PDA to be invisible
-put on invisible ID
-it's visible anyways?

Antag clowns being stuck with clumsiness is already bad enough and should be removed, now we can't even sneak in anywhere without playing ID in pocket games.
![screenshot_110](https://user-images.githubusercontent.com/5169310/34463549-5f46c9b8-ee2d-11e7-94f7-76edc8082e53.png)

**Comment by Dacendeth** [comment]
Just put it in a wallet, wallets don't have sprites.

**Comment by Killette2** [comment]
and interact with vox? no thanks

**Comment by Ephx1** [comment]
> Vox are the only way to get wallets

Get a load of this guy

**Comment by Killette2** [comment]
yeah let me take time out of my round to explore deffic maint or grow a ton of shit in botany when I just want to be an invisible clown

**Comment by Ephx1** [comment]
> He can't ask nicely for a wallet

Get a load of this guy

**Comment by Dacendeth** [comment]
You can buy wallets from cargo merch computer..

**Comment by Killette2** [comment]
:s clown's buying a wallet at cargo. Get some t-ray scanners and keep camera tracked on him
